src/main/java/io/confluent/connect/jdbc/source/JdbcSourceConnectorConfig.java

```diff
@@ -330,6 +336,29 @@
       + "  In most cases it only makes sense to have either TABLE or VIEW.";
   private static final String TABLE_TYPE_DISPLAY = "Table Types";
 
+  public static final String TRANSACTION_ISOLATION_MODE_DEFAULT =
+          TransactionIsolationMode.DEFAULT.name();
+  public static final String TRANSACTION_ISOLATION_MODE_CONFIG = "transaction.isolation.mode";
+  private static final String TRANSACTION_ISOLATION_MODE_DOC =
+          "Mode to control which transaction isolation level is used when running queries. "
+                  + "against the database. By default no explicit transaction isolation"
+                  + "mode is set. SQL_SERVER_SNAPSHOT_ISOLATION_MODE will only work"
+                  + "against a connector configured to write to Sql Server. "
+                  + "The Microsoft Sql Server driver is a requirement to use this feature. "
```

Comment:
I don't think we need to explicitly state anything about the SQL server driver requirement. Those using the connector against non-SQL server database would not expect `SQL_SERVER_SNAPSHOT_ISOLATION_MODE` to work and would not be required to have the SQL server driver to use any other isolation modes.